src/regexbench.cpp

```diff
@@ -225,7 +228,7 @@ Arguments parse_options(int argc, const char *argv[]) {
   po::options_description optargs("Options");
   optargs.add_options()("help,h", "Print usage information.");
   optargs.add_options()(
-      "engine,e", po::value<std::string>(&engine)->default_value("hyperscan"),
+      "engine,e", po::value<std::string>(&engine)->default_value("rematch"),
```

Comment:
Wouldn't it be better to leave the default engine as one of the open-source matcher? Since regexbench itself is an open-source project, it doesn't sound right to require proprietary software by default.